draft-ietf-rtcweb-jsep.xml

```diff
@@ -2263,10 +2263,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         deserialized information. However, SDP allows many types of lines, not
         all of which are relevant to JSEP applications.
         For each line, the implementation will first ensure it is syntactically
-        correct according its defining ABNF [TODO: reference], check
+        correct according its defining ABNF, check
         that it conforms to <xref target="RFC4566"/> and <xref target="RFC3264"/>
         semantics, and then either parse and store or discard the provided value,
-        as described below.
+        as described below. A partial list of ABNFs for SDP can found in
+	Section 9 of <xref target="RFC4566"/>,
```

Comment:
It would be nice to display this in tabular format, e.g. attribute in one column, and ABNF reference in a second column.